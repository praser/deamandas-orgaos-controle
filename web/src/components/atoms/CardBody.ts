import styled from 'styled-components';

const CardBody = styled.div`
  /* Layout */
  box-sizing: border-box;
  display: block;
  flex-basis: auto;
  flex-grow: 1;
  flex-shrink: 1;
  min-height: 1px;
  padding-bottom: 20px;
  padding-left: 20px;
  padding-right: 20px;
  padding-top: 20px;
`;

export default CardBody;
