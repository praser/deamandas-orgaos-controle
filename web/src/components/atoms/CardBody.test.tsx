import React from 'react';
import { render, screen } from '@testing-library/react';
import CardBody from './CardBody';

describe('CardBody component', () => {
  let cardbody: HTMLElement;
  const testId = 'testid';

  beforeEach(() => {
    render(<CardBody data-testid={testId} />);
    cardbody = screen.getByTestId(testId);
  });

  it('is expected to match the snapshot', () => {
    expect(cardbody).toMatchSnapshot();
  });
});
