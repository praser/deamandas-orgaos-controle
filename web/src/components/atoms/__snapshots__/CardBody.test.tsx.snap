// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardBody component is expected to match the snapshot 1`] = `
<div
  class="sc-bdvvaa ivgyQo"
  data-testid="testid"
/>
`;
